pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -136,6 +136,12 @@ const (
 	ReasonResourceVerificationFailed = "ResourceVerificationFailed"
 )
 
+const (
+	taskRun   = "TaskRun"
+	customRun = "CustomRun"
+	run       = "Run"
+)
+
```

Comment:
we already have these constants defined in https://github.com/tektoncd/pipeline/blob/90f33a63497e8c07fa799f8cab5138756a075299/pkg/apis/pipeline/controller.go#L19-L42

Reply:
Is it possible to reuse them here instead of redefining the constants?

Reply:
Thanks, I didn't know about these!